client/src/app/main-page/main-page.component.scss

```diff
@@ -0,0 +1,19 @@
+.container {
+  padding: 30px;
+  height: 100vh;
+  background: #EFEFEF;
+  position: relative;
+
+  .wrapper {
+    display: flex;
+    height: calc(100% - 50px);
```

Comment:
Don't understand this calculation

Reply:
the 50px is height of the header, the height of wrapper is total height subscribe header height